docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -622,6 +622,33 @@ metrics:
           extract_value: '([a-zA-Z0-9_]+)' # will ignore surrounding non-printable characters
 ```
 
+#### Report states
```

Comment:
```suggestion
#### Apply mapping to OID value in `metric_tags`
```

I think we can be more generic here. Mapping `states` is only one specific case, but we can map any value to another value.

Reply:
My goal was to document how to use constant metrics to report string OIDs, `Apply mapping to OID value in metric_tags` is already documented [here](https://github.com/DataDog/integrations-core/blob/9259684c3047a7100f6f17aaee27cb6820895e75/docs/developer/tutorials/snmp/profile-format.md?plain=1#L278-L301) I think
But I agree the title is not generic enough, maybe something like `Report string OIDs`?